app/routes.js

```diff
@@ -1,14 +1,13 @@
 // @flow
 import React from 'react';
-import { Route, IndexRoute } from 'react-router';
+import { Route } from 'react-router';
 import App from './containers/App';
 import HomePage from './containers/HomePage';
 import CounterPage from './containers/CounterPage';
 
-
 export default (
   <Route path="/" component={App}>
-    <IndexRoute component={HomePage} />
+    <Route path="/" component={HomePage} />
```

Comment:
@amilajack I opened a PR on the react-router-4 branch, see https://github.com/chentsulin/electron-react-boilerplate/pull/901